src/components/menus/Menu.svelte

```diff
@@ -129,7 +128,11 @@
     role="menu"
     use:popperRef
     on:click|stopPropagation={onClick}
-    on:mouseenter={() => hideAllTooltips()}
+    on:mouseenter={() => {
+      // TODO why is this necessary? Also destroys tooltips and they don't show up
+      // after menus are opened if they were previously shown.
+      // hideAllTooltips();
+    }}
```

Comment:
Not sure if this actually happens currently, but I could see this being for the case where a tooltip is opened via non hover interactions, and potentially the menu popping up under the tooltip.

Reply:
Can confirm that this is achievable:
![Screenshot 2023-12-04 at 5 00 29 PM](https://github.com/NASA-AMMOS/aerie-ui/assets/5290214/6d9b8cd8-d3d7-4db0-846b-e942dc88f798)
To replicate:
1. I clicked on the white space to the left of the pan/zoom toggle button and then pressed tab to highlight it
2. Then I hovered over the menu directly above it

Reply:
Ah interesting, good find. I'll make a separate issue to track the underlying issue where closing tooltips from within the modal util prevents them from opening again.